source/core/TypeAndDataPathFinder.cpp

```diff
@@ -0,0 +1,244 @@
+/**
+
+Copyright (c) 2015 National Instruments Corp.
+
+This software is subject to the terms described in the LICENSE.TXT file
+
+SDG
+*/
+
+/*! \file
+ */
+
+#include "TypeAndDataManager.h"
+
+namespace Vireo
+{
+
+//------------------------------------------------------------
+/* Create a visitor with a needle being looked for and a TypeRef
+* that describes what the needle is a pointer to. The visitor will visit
+* all types in the type manger and thus ultimate all values owned by that type manager
```

Comment:
Fix typo "manger" to manager is this file and the where this file was copied from.